### PR TITLE
Feature/issue 152 ignore own events

### PR DESCRIPTION
**Pull request by artem-azarov**
Issue #152

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -247,6 +256,10 @@ class Background {
     }
     return message;
   }
+
+  fetchCurrentUser() {
+    return this.currentUser ? Promise.resolve(this.currentUser) : this.gitlab.getCurrentUser();
```
Comment (artem-azarov) [inline src/background.js]: **help wanted**: not sure is this type of dummy cache is acceptable, probably I should add smthng like `BackgroundCache` and keep the current user instance in the storage?

Comment (artem-azarov) [reply src/background.js]: make sense, thx. 

... also this approach reduced the number of changes

File: src/options.js [hunk 2]
```diff
@@ -36,10 +36,16 @@ app.view = function(vnode) {
       });
     }
 
-    state.saveConfig(state, projects);
+    // get and save user_id when API is available
+    const hasAuthParams = state.api_path !== "" && state.private_token !== "";
+    const userIdPromise = hasAuthParams ? state.gitlab.getCurrentUser().then(({ id }) => id) : Promise.resolve("");
```
Comment (sue445) [inline src/options.js]: 🆒

**Comment by sue445** [comment]
I released as `v2.5.0` but currently under reviewing

**Comment by artem-azarov** [comment]
released :) thx a lot!

**Pull request merged**